hadoop-tools/hadoop-distcp/src/test/java/org/apache/hadoop/tools/TestDistCpOptions.java

```diff
@@ -225,6 +225,45 @@ public void testSetDeleteMissing() {
     }
   }
 
+  @Test
+  public void testDeleteMissingUseTrash() {
```

Comment:
All assert true/false calls *MUST* have a string message to include on a failure. 

Imagine you've got a report of a test run failure on jenkins, you want to understand what has failed without trying to work out exactly which line does what on that specific branch of the code